src/Domain/LeanCode.CQRS/IRemoteOperation.cs

```diff
@@ -0,0 +1,4 @@
+namespace LeanCode.CQRS
+{
+    public interface IRemoteOperation<out TResult> : IOperation<TResult> { }
```

Comment:
Let's get rid of it. We don't need the `IRemoteT`/`IT` distinction. No one uses it anymore, really.

Reply:
Do you mean, for all types?

Reply:
In this PR only for this one. For v6 - I would remove all of them.